### DISCUSSION TITLE
Perma no credit.

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#82] [Ideas]
Just a suggestion to permanently check for credit change and remove them the instant someone tries to add credit with mod/cheats.

**Comment by winstxnhdw** [comment]
No idea how I should be implementing this. I assume you still want to earn credits from selling.

**Reply by yujispin990** [reply]
nope 0 like somehow it checks if any credit is added and just removes them immediately. But i understand if its not feasible.

**Reply by winstxnhdw** [reply]
I mean if it's just any amount of credit. It's possible. It's just hard if you want to make an exception for selling scraps.

**Reply by yujispin990** [reply]
hmm, is it possible to make it toggleable? like a switch like /nocredit 1 for true and /nocredit 0 for false.

**Reply by winstxnhdw** [reply]
Sure.

**Reply by yujispin990** [reply]
if its too much its ok, it was just a suggestion im very happy that you even acknowledge my suggestions :D

**Reply by winstxnhdw** [reply]
I have added this too. You can try it by doing `/block credits`. I haven't tested it yet though. You can try it first and let me know.
